com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/resolvers/v3_1/DescriptorPlaceholderResolver.java

```diff
@@ -0,0 +1,44 @@
+package com.sap.cloud.lm.sl.mta.resolvers.v3_1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.sap.cloud.lm.sl.common.ContentException;
+import com.sap.cloud.lm.sl.mta.model.SystemParameters;
+import com.sap.cloud.lm.sl.mta.model.v2_0.Platform;
+import com.sap.cloud.lm.sl.mta.model.v2_0.Target;
+import com.sap.cloud.lm.sl.mta.model.v3_1.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.model.v3_1.Module;
+import com.sap.cloud.lm.sl.mta.resolvers.ResolverBuilder;
+
+public class DescriptorPlaceholderResolver extends com.sap.cloud.lm.sl.mta.resolvers.v2_0.DescriptorPlaceholderResolver {
+
+    protected final DeploymentDescriptor deploymentDescriptor3_1;
```

Comment:
I think, it can be defined just as "deploymentDescriptor" so it will "hide" deploymentDescriptor from com.sap.cloud.lm.sl.mta.resolvers.v2_0.DescriptorPlaceholderResolver